files/build_templates/radv.service.j2

```diff
@@ -7,7 +7,7 @@ StartLimitIntervalSec=1200
 StartLimitBurst=3
 
 [Service]
-User={{ sonicadmin_user }}
+User=root
```

Comment:
I am not sure this is a good change. @jleveque are you familiar with this part? The file above is also setting user to root directly.

Reply:
Good catch. There is no `admin` user inside our containers, only `root`. This will not work. What is the reason for attempting to change the user here?

Reply:
As discussed offline, it should be OK.

Reply:
Please remove this change above.

Reply:
Fixed. Thanks!